src/styles/typeTokens.js

```diff
@@ -1,108 +1,8 @@
-const FONT_TYPES = ['body', 'headings', 'interface'].reduce(
-  (accumulator, type) => {
-    return { ...accumulator, [type]: type };
-  },
-  {},
-);
-
-// In order to get a whole-integer pixel number, we multiply by ten and round,
-// then divide by ten again to get a rem value that works with our base size setting.
-const calculateFontSize = ({ scaleModifier, sizeNumber, baseSize }) => {
-  const newSize = Math.round(baseSize * 10 * scaleModifier ** sizeNumber) / 10;
-
-  return newSize;
-};
-
-// Calculate line height for either single-line (heading) or multi-line (body) usage.
-const calculateLineHeight = ({ fontSize, targetLineHeight }) => {
-  // Check if the `fontSize * lineHeight` is divisible by 4. If not, increment
-  // until it is divisible by four. This ensures that line-height respects our
-  // vertical rhythm.
-  let targetLineHeightInPixels = Math.ceil(targetLineHeight * fontSize);
-
-  // Increment the target line height until it's a multiple of 4.
-  while (targetLineHeightInPixels % 4 !== 0) {
-    targetLineHeightInPixels += 1;
-  }
-
-  // Generate a unitless line height relative to the font size that corresponds
-  // with our target pixel height.
-  const lineHeight = targetLineHeightInPixels / fontSize;
-  return lineHeight;
-};
-
-// Convert regular case to small-caps.
-const smallCaps = () => {
-  return {
-    letterSpacing: '0.05em',
-    textTransform: 'uppercase',
-  };
-};
-
-// Output typographic styling to apply to elements.
-const typeAttributes = ({ theme, sizeNumber, fontType }) => {
-  const fontSize = calculateFontSize({
-    baseSize: theme.typography.baseSizes.desktop,
-    scaleModifier: theme.typography.scaleModifiers.desktop,
-    sizeNumber,
-  });
-
-  // Get the target line height from our theme for the font used.
-  const targetLineHeight = theme.typography.lineHeights[fontType];
-
-  return {
-    fontSize: `${fontSize}rem`,
-    lineHeight: calculateLineHeight({
-      fontSize: fontSize * 10,
-      targetLineHeight,
-    }),
-  };
-};
-
-// Interface
-export const interfaceSmall = (theme) => {
-  return {
-    fontFamily: theme.typography.fonts.interface,
-    fontWeight: theme.typography.fontWeights.interface,
-    ...typeAttributes({
-      theme,
-      sizeNumber: -0.5,
-      fontType: FONT_TYPES.interface,
-    }),
-  };
-};
-
-export const interfaceMedium = (theme) => {
-  return {
-    fontFamily: theme.typography.fonts.interface,
-    fontWeight: theme.typography.fontWeights.interface,
-    ...typeAttributes({
-      theme,
-      sizeNumber: 0,
-      fontType: FONT_TYPES.interface,
-    }),
-  };
-};
-
-export const interfaceLarge = (theme) => {
-  return {
-    fontFamily: theme.typography.fonts.interface,
-    fontWeight: theme.typography.fontWeights.interface,
-    ...typeAttributes({
-      theme,
-      sizeNumber: 1,
-      fontType: FONT_TYPES.interface,
-    }),
-  };
-};
+import { FONT_TYPES } from './constants';
```

Comment:
Question: what is a font type here?

Reply:
They're referencing values in themes, but it's all a bit hard-to-follow right now. I'm gonna see if I can improve it a bit...